### PR TITLE
[tx_hash label] [Easy] Add bluechip_investment tx_hash label

### PR DESCRIPTION
**Pull request by gentrexha**
Brief comments on the purpose of your changes:

Implement the blue-chip investment tx hash label

cc @bh2smith for internal review

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### CODE REVIEW

File: models/cow_protocol/tx_hash_labels/bluechip_investment/cow_protocol_tx_hash_labels_bluechip_investment.sql
Comment (bh2smith) [inline models/cow_protocol/tx_hash_labels/bluechip_investment/cow_protocol_tx_hash_labels_bluechip_investment.sql:8]: nit:
```suggestion
SELECT * FROM {{ ref('cow_protocol_tx_hash_labels_bluechip_investment_ethereum') }}

```

**Review by bh2smith — APPROVED** [review]
LGTM!

File: models/cow_protocol/tx_hash_labels/bluechip_investment/cow_protocol_tx_hash_labels_bluechip_investment.sql
Comment (gentrexha) [reply models/cow_protocol/tx_hash_labels/bluechip_investment/cow_protocol_tx_hash_labels_bluechip_investment.sql:8]: I can't seem to spot the difference between the two. Can you elaborate on that?

**Comment by gentrexha** [comment]
This will be fixed once #2748 is merged.

File: models/cow_protocol/tx_hash_labels/bluechip_investment/cow_protocol_tx_hash_labels_bluechip_investment.sql
Comment (bh2smith) [reply models/cow_protocol/tx_hash_labels/bluechip_investment/cow_protocol_tx_hash_labels_bluechip_investment.sql:8]: New line at EOF!

File: models/cow_protocol/tx_hash_labels/bluechip_investment/ethereum/cow_protocol_tx_hash_labels_bluechip_investment_ethereum.sql
Comment (bh2smith) [inline models/cow_protocol/tx_hash_labels/bluechip_investment/ethereum/cow_protocol_tx_hash_labels_bluechip_investment_ethereum.sql:16]: Could you add comment strings identifying the token symbols here?

**Review by bh2smith — APPROVED** [review]

**Comment by jeff-dude** [comment]
@gentrexha i think we'll need to wait for prod to refresh with #2748 being merged. should be within the hour.

**Comment by jeff-dude** [comment]
we're past the merge issues, but now have failure on unique test [here](https://github.com/duneanalytics/spellbook/blob/main/models/cow_protocol/tx_hash_labels/cow_protocol_tx_hash_labels_all_schema.yml#L6) -- please do investigate and fix. it'd also be better to move these unique test to the base spells which contain the logic, rather than overall view, to catch it sooner.

**Comment by gentrexha** [comment]
> we're past the merge issues, but now have failure on unique test [here](https://github.com/duneanalytics/spellbook/blob/main/models/cow_protocol/tx_hash_labels/cow_protocol_tx_hash_labels_all_schema.yml#L6) -- please do investigate and fix. it'd also be better to move these unique test to the base spells which contain the logic, rather than overall view, to catch it sooner.

Thanks for taking a look into these.

I've removed the test due to it not being valid anymore because we are okay with the same tx_hash showing across multiple different projects/trades with CoWs. cc @bh2smith The thing we discussed where the same tx_hash being shown twice started happening.

**Pull request merged**
